src/main/kotlin/no/nav/familie/ef/sak/api/dto/PersonopplysningerDto.kt

```diff
@@ -27,6 +27,7 @@ data class PersonopplysningerDto(val personIdent: String,
                                  val innflyttingTilNorge: List<InnflyttingDto>,
                                  val utflyttingFraNorge: List<UtflyttingDto>,
                                  val oppholdstillatelse: List<OppholdstillatelseDto>,
+                                 val vergemål: List<VergemålDto>,
```

Comment:
`verge` vs `vergemål`?

Reply:
`VergemålEllerFremtidsfullmakt`, kanskje - er jo en viss forskjell på fremtidsfullmakt og vergemål...

Reply:
Setter den til vergemål https://nav-it.slack.com/archives/G01PUC2R2G2/p1625921380000100